### PR TITLE
sc-61: Sc 61/homepage

### PR DESCRIPTION
**Pull request by connie-ho**
Summary:
- add load NFTs function
- lint previously unlinted files 
- fix lint staged
- fix an error in ethers provider
- update app test to accept ethers provider

TODO Future:
1. Style homepage once create functionality is merged
2. Hide ropsten project URL in github secrets
3. Write test for the homepage once 1 is done

### CODE REVIEW

File: test/sample-test.js [hunk 1]
```diff
@@ -1,81 +1,87 @@
 /* test/sample-test.js */
```
Comment (connie-ho) [inline test/sample-test.js:1]: this is just the linter applied

**Comment by connie-ho** [comment]
If someone knows a way to not commit ABIs that would be lovely

File: frontend/src/hooks/use-wallet-connection.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
-import {useState, useEffect} from "react";
+import { useState, useEffect } from 'react';
```
Comment (connie-ho) [inline frontend/src/hooks/use-wallet-connection.js:1]: linter stuff

File: frontend/src/contexts/ethers-provider-context.js [hunk 3]
```diff
@@ -1,27 +1,20 @@
...
-  const marketContract = new ethers.Contract(nftmarketaddress, Market.abi, signer);
-
-  const value = {
-    provider,
-    signer,
-    tokenContract,
-    marketContract,
-  };
+  const provider = new ethers.providers.JsonRpcProvider();
+  // const provider = new ethers.providers.getDefaultProvider(process.env.REACT_APP_PROJECT_URL);
```
Comment (connie-ho) [inline frontend/src/contexts/ethers-provider-context.js:12]: kept these comments in case we want to use a different provider later

**Review by Justin1002 — APPROVED** [review]

**Pull request merged**